src/v2/Apps/Conversation/Components/ConfirmArtworkButton.tsx

```diff
@@ -2,65 +2,62 @@ import { useState } from "react"
 import * as React from "react"
 import { createFragmentContainer, graphql, RelayProp } from "react-relay"
 import { useTracking } from "react-tracking"
-import { Button } from "@artsy/palette"
+import { Button, ButtonProps } from "@artsy/palette"
 import { TappedConfirmArtwork } from "@artsy/cohesion"
 
 import createLogger from "v2/Utils/logger"
 import { MakeInquiryOffer } from "../Mutation/MakeInquiryOfferMutation"
+import { MakeInquiryOrder } from "../Mutation/MakeInquiryOrderMutation"
 
 import { ConfirmArtworkButton_artwork } from "v2/__generated__/ConfirmArtworkButton_artwork.graphql"
 
 const logger = createLogger("Conversation/Components/ConfirmArtworkButton.tsx")
 
-export interface ConfirmArtworkButtonProps {
+export interface ConfirmArtworkButtonProps extends ButtonProps {
```

Comment:
I needed to add `variant` as well, so it was a good opportunity to replicate the job made on Eigen.